src/rtl_433.c

```diff
@@ -1331,6 +1331,22 @@ int main(int argc, char **argv) {
         r_free_cfg(cfg);
         exit(!r);
     }
+
+    // Special case for test pattern demo mode
+    if (cfg->demo_mode) {
+        r = 0;
+        for (void **iter = demod->r_devs.elems; iter && *iter; ++iter) {
+            r_device *r_dev = *iter;
+            if (cfg->verbosity)
+                fprintf(stderr, "Showing demo data for device %s.\n", r_dev->name);
```

Comment:
If all devices are enabled, does this show 'Showing demo data' for each device? This doesn't seem right to me, as most devices (currently) wouldn't have any test patterns.

Reply:
Yes, that should be considered. Anyway, I'll let this PR sit until I have a script to import patterns for most of the devices from the test repo. Won't be too much an issue then.